resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,1797 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders API
+  contact: {}
+  version: '2.0'
+  description: |
+    # Overview
+    Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS)
+    and to get messages sent from customs offices of departure and destination.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+    variables: {}
+paths:
+  /customs/transits/movements/departures:
+    post:
+      tags:
+      - Departures
+      summary: Send a declaration data message
+      description: >
+        Send a transit declaration to the Customs Office of Departure to notify it that you wish to carry out a transit operation.
+
+        This request is also called an ‘Declaration Data’ E_DEC_DAT (IE015) message.
+
+        A successful response will contain a URI which you can use to get the message status. A successful response can also include an optional box ID, which you can use to match this movement to any messages received from the Push Pull Notifications API.
+         <br/><br/>
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: postDeclarationDataMessage
+      parameters:
+        - $ref: '#/components/parameters/acceptHeader'
+        - $ref: '#/components/parameters/contentXmlJsonHeader'
+      responses:
+        '202':
+          description: A successful departure declaration response
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsDeparturesResponse'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/departures/62fb8f33d233578e
+                      messages:
+                        href: /customs/transits/movements/departures/62fb8f33d233578e/messages
+                    boxId: b54c2424-955b-496c-bbce-dfb9f57cd797
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/departures/62fb8f33d233578e
+                  messages:
+                    href: /customs/transits/movements/departures/62fb8f33d233578e/messages
+                boxId: b54c2424-955b-496c-bbce-dfb9f57cd797
+        '413':
+          description: Request Entity Too Large
+          content:
+            application/json:
+              schema:
+                allOf:
+                 - $ref: '#/components/schemas/RequestEntityTooLarge'
+                 - example:
+                    code: REQUEST_ENTITY_TOO_LARGE
+                    message: Your message size must be less than 500000 bytes

```

Comment:
Is `500000 bytes` correct? That approximates to 0.47MB. I thought that the threshold for small messages was originally 0.5MB. Of course, the current thinking is that small messages will be less than 5MB and large messages will be between 5MB and 22MB.

Also, which works best for units of measurement - bytes or MB?

Reply:
So, 500000 bytes is actually 0.5MB, and is approximately 0.47**MiB**. There has been a push to standardise MB as a base 1000 modiifer, and MiB as a base 1024 modifier -- see https://en.wikipedia.org/wiki/Byte#Multiple-byte_units.

So this is actually 0.5MB (now).

Reply:
I ran the rest api with more than 5 MB and got this error so have mentioned the same message here as well.
@fmcgrath / @drnaylor-oc  do we need to change it to something else?

Reply:
It won't run with 5MB yet, so it's of no surprise. However, I'd just stick another 0 on the end of the definition, ready for when we do accept 5MB.